swagger/src/main/scala/org/http4s/rho/swagger/models.scala

```diff
@@ -196,18 +196,53 @@ object models {
     , delete           : Option[Operation] = None
     , patch            : Option[Operation] = None
     , options          : Option[Operation] = None
+    , head             : Option[Operation] = None
     , parameters       : List[Parameter]   = Nil
     , vendorExtensions : Map[String, Any]  = Map.empty
     ) {
-
+    
+    import com.fasterxml.jackson.annotation.{ JsonPropertyOrder, JsonIgnore }
+    
+    @JsonPropertyOrder(Array("get", "post", "put", "delete", "options", "patch", "head"))
+    class ComplementaryPath extends jm.Path {
```

Comment:
I think we just need to bump our swagger-core dep to 1.5.3 and we get these `head` operations, right? (See [here](https://github.com/swagger-api/swagger-core/blob/v1.5.3/modules/swagger-models/src/main/java/io/swagger/models/Path.java#L54))